tools/requirements.txt

```diff
@@ -6,6 +6,6 @@ pypsrp[kerberos,credssp]
 pywinrm[kerberos,credssp]
 toml
 pexpect>=4.5
-python-daemon
+fasteners>=0.16
```

Comment:
@Shrews @eqrx not sure if I need to edit this file.

Reply:
I'm not sure what this file is for. Perhaps check with @shanemcd